src/service/resources-impl.js

```diff
@@ -712,6 +712,7 @@ export class ResourcesImpl {
         dict({
           'title': doc.title,
           'sourceUrl': getSourceUrl(this.ampdoc.getUrl()),
+          'isStory': doc.body.firstElementChild.tagName === 'AMP-STORY',
```

Comment:
There's no guarantee that the `firstElementChild` exists (an empty body and textual only body are both valid).
```suggestion
          'isStory': doc.body.firstElementChild?.tagName === 'AMP-STORY',
```